pkgs/applications/misc/gnome-epub-thumbnailer/default.nix

```
@@ -0,0 +1,50 @@
+{ stdenv
+, lib
+, fetchurl
+, wrapGAppsNoGuiHook
+, meson
+, ninja
+, pkg-config
+, gnome
+, gdk-pixbuf
+, libarchive
+, libxml2
+}:
+
+stdenv.mkDerivation rec {
+  pname = "gnome-epub-thumbnailer";
+  version = "1.7";
+
+  src = fetchurl {
+    url = "mirror://gnome/sources/gnome-epub-thumbnailer/${lib.versions.majorMinor version}/${pname}-${version}.tar.xz";
```

```suggestion
    url = "mirror://gnome/sources/gnome-epub-thumbnailer/${lib.versions.majorMinor version}/gnome-epub-thumbnailer-${version}.tar.xz";
```

to make it less annoying when overriding `src`.